docs/source/linkintegrity.rst

```diff
@@ -0,0 +1,19 @@
+Link integrity
+==============
+
```

Comment:
I'd like to have a short introduction of how linkintegrity works in Plone in general.

Reply:
added a short introduction, let me know if it's ok or need to be improved